data/templates/frr/daemons.frr.tmpl

```diff
@@ -11,47 +24,82 @@ ldpd=yes
 nhrpd=no
 eigrpd=yes
 babeld=yes
-sharpd=no
+sharpd=yes
 pbrd=no
 bfdd=yes
-staticd=yes
+fabricd=yes
+vrrpd=no
+pathd=yes
```

Comment:
Why do we need to start pathd?

Reply:
https://docs.frrouting.org/en/stable-9.0/pathd.html

PATH is a daemon that handles the installation and deletion of Segment Routing (SR) Policies. Based on MPLS (This means that your OS of choice must support MPLS), SR add a stack of MPLS labels to ingress packets so these packets are egress through the desired path.

The SR policies and Segment Lists can be configured either locally by means of vtysh or centralized based on a SDN controller (ODL, Cisco, …) communicating using the PCEP protocol (RFC 5440).

https://tools.ietf.org/html/rfc5440.html

Reply:
We should not enable daemons that VyOS is not using / has no CLI

Reply:
CLI exists through `vtysh`.

If FRR wasnt modularized these modules would already have been enabled.

Only modules/features of FRR who shall NOT be enabled are those who are replaced by other packages (pimd, nhrpd, pbrd and vrrpd).

Please let me know if you still think that fabricd and pathd should be disabled?

Reply:
As long as there is no VyOS CLI this should be disabled

Reply:
Commit updated by removing run of fabricd and pathd.